tests/unit/test_runtime.py

```diff
@@ -306,7 +306,9 @@ async def test_simple_browse(temp_dir, box_class):
     runtime = await _load_runtime(temp_dir, box_class)
 
     # Test browse
-    action_cmd = CmdRunAction(command='python -m http.server 8000 > server.log 2>&1 &')
+    action_cmd = CmdRunAction(
+        command='$OPENDEVIN_PYTHON_INTERPRETER -m http.server 8000 > server.log 2>&1 &'
```

Comment:
`$OPENDEVIN_PYTHON_INTERPRETER` may not be available in `EventStreamRuntime`

Reply:
In future? The test are passed now. Python is only not available in stream runtime

Reply:
![image](https://github.com/user-attachments/assets/dc6cbd06-9a85-496a-b38c-a8ea3987cfc4)
[link](https://github.com/OpenDevin/OpenDevin/actions/runs/10138446082/job/28030667691?pr=3156#step:8:1641)

Reply:
Yes, `$OPENDEVIN_PYTHON_INTERPRETER` is not available in `EventStreamRuntime`
and `python` is not available in `ServerRuntime`